tests/rpc/src/common_parameters.js

```diff
@@ -10,35 +10,227 @@ var handlersToClean = [];
+// ##################### //
+
+var pointGeojsonObject = {
+  'type': 'FeatureCollection',
+  'crs': {
+    'type': 'name',
+    'properties': {
+      'name': defaultPosition.srsName
+    }
+  },
+  'features': [{
+    'type': 'Feature',
+    'geometry': {
+      'type': 'Point',
+      'coordinates': [defaultPosition.centerX, defaultPosition.centerY]
+    },
+    'properties': {
+      'label': 'I am a point feature!',
+      'test_property': "point feature"
+    }
+  }]
 };
 
+var addPointFeatureParams = [pointGeojsonObject, {
+  layerId: 'VECTOR',
+  clearPrevious: false,
+  centerTo: true,
+  cursor: 'zoom-in',
+  featureStyle: {
+    image: {
+      shape: 4,
+      size: 5,
+      color: '#ff3300',
+      stroke: '#000000'
+    },
+    text: {
+      scale: 1.3,
+      fill: {
+        color: 'rgba(0,0,0,1)'
+      },
+      stroke: {
+        color: 'rgba(255,255,255,1)',
+        width: 2
+      },
+      labelProperty: 'label',
+      offsetX: 65,
+      offsetY: 8
+    }
+  }
+}];
+
+var lineGeojsonObject = {
+  'type': 'FeatureCollection',
+  'crs': {
+    'type': 'name',
+    'properties': {
+      'name': defaultPosition.srsName
+    }
+  },
+  'features': [{
+    'type': 'Feature',
+    'geometry': {
+      'type': 'LineString',
+      'coordinates': [
+        [defaultPosition.centerX, defaultPosition.centerY],
+        [defaultPosition.centerX + 100000, defaultPosition.centerY + 1100000]
+      ]
+    },
+    'properties': {
+      'label': "I am a line feature!",
+      'test_property': "line feature"
+    }
+  }]
+};
+
+var addLineFeatureParams = [lineGeojsonObject, {
+  layerId: 'VECTOR',
+  clearPrevious: true,
+  layerOptions: testLayerOptions,
+  centerTo: true,
+  cursor: 'zoom-in',
+  featureStyle: {
+    fill: {
+      color: '#2200ff'
+    },
+    stroke: {
+      color: '#2200ff',
+      width: 3
+    },
+    text: {
+      scale: 2.0,
+      fill: {
+        color: 'rgba(2,2,0,1)'
+      },
+      stroke: {
+        color: 'rgba(0,255,1,1)',
+        width: 0
+      },
+      labelProperty: 'label'
+    }
+  },
+  prio: 4,
+  //minScale: 1451336
+}];
+
+// Styling for object
+var featureStyle = {
+  fill: {
+    color: 'rgba(0,0,0,0.3)',
+  },
+  stroke: {
+    color: '#FF0000',
+    width: 5
+  },
+  text : {
+    scale : 2,
+    fill : {
+      color : 'rgba(0,0,0,1)'
+    },
+    stroke : {
+      color : 'rgba(255,255,255,0)',
+      width : 1
+    },
+    labelText: 'Test feature'
+  }
+};
+
+var testLayerOptions = {
+  'minResolution': 0,
+  'maxResolution': 10000
+};
+
+// # Requests: Change map layer style
+var styleParams = {
+  SLD_BODY: '<StyledLayerDescriptor xmlns=\"http://www.opengis.net/sld\" xmlns:ogc=\"http://www.opengis.net/ogc\" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0.0" xsi:schemaLocation="http://www.opengis.net/sld StyledLayerDescriptor.xsd">' +
+    '    <NamedLayer>' +
+    '    <Name>oskari:kunnat2013</Name>' +
```

Comment:
I guess the name doesn't really have any meaning here?

Reply:
Doesn't seem like a reasonable default so maybe use some other name or document if tests assume a layer with this name.

Reply:
Will check.